src/components/AccountConfigDialog/DisconnectAccountConfigDialog.jsx

```
@@ -0,0 +1,594 @@
+          <Text size="small" weight="regular" color="#414651">
+            {`Disconnecting your cloud account will disconnect all associated
+            instances from the ${serviceOrgName} Control Plane`}
+          </Text>
+        </ListItem>
+      </List>
+      <Box>
+        <Text
+          size="small"
+          weight="semibold"
+          color="#344054"
+          dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)